draft-ietf-quic-tls.md

```diff
@@ -760,14 +756,12 @@ and ignoring any outstanding Initial packets.
 
 ### Discarding Handshake Keys
 
-An endpoint MUST NOT discard its handshake keys until the TLS handshake is
-confirmed ({{handshake-confirmed}}).  An endpoint SHOULD discard its handshake
-keys as soon as it has confirmed the handshake.  Most application protocols
-will send data after the handshake, resulting in acknowledgements that allow
-both endpoints to discard their handshake keys promptly.  Endpoints that do
-not have reason to send immediately after completing the handshake MAY send
-ack-eliciting frames, such as PING, which will cause the handshake to be
-confirmed when they are acknowledged.
+An endpoint MUST discard its handshake keys when the TLS handshake is confirmed
+({{handshake-confirmed}}).  The server MUST send a HANDSHAKE_DONE frame when the
+handshake completes.  Most application protocols will send data after the
+handshake.  A server might want to minimize the number of packets sent by
+bundling the HANDSHAKE_DONE frame with other data.  It MAY delay sending of the
+HANDSHAKE_DONE frame by no more than one round-trip.
```

Comment:
This delay sentence is somewhat in conflict with the preceding requirement, so maybe instead say "MUST send within 1 round-trip time of handshake completion."

Reply:
Done.